parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
```

Comment:
@pateljm Right now the travis in apache runs on a older version of FLEX. Hence it uses the files present in parser/preprocessed by default.  Any one who adds new tokens/rules to the parser has to run parser/genfiles.sh  and check the generated files in  for the changes to work.  We should be making sure that this is a part of the documentation for the parser workflow.